contracts/orochi/XOroV2.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.19;
+
+import '@openzeppelin/contracts/token/ERC1155/ERC1155.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+error AccessDenied();
+
+contract XOroV2 is ERC1155, Ownable {
+  uint8 public constant TOKEN_ID = 1;
+
+  modifier accessDenied() {
+    revert AccessDenied();
+    _;
+  }
+
+  constructor() ERC1155('https://metadata.orochi.network/x-oro-v2/{id}.json') {}
+
+  function mint(address to, uint256 amount) public onlyOwner {
+    _mint(to, TOKEN_ID, amount, '');
+  }
+
+  function batchMint(uint256[] calldata packedData) public onlyOwner {
+    for (uint i = 0; i < packedData.length; i += 1) {
+      (uint96 amount, address beneficiary) = _unpack(packedData[i]);
+      mint(beneficiary, amount);
+    }
+  }
+
+  function balance(address account) public view virtual returns (uint256) {
```

Comment:
Is `balance()` a correct method?

Reply:
At the begining, I add this function to delegate balanceOf without adding TOKEN_ID parameter. But now I think it is unnecessary. Deleted.